docs/general/configuration_options.md

```diff
@@ -5,6 +5,7 @@ The following configuration options can be set on `ActiveModel::Serializer.confi
 ## General
 
 - `adapter`: The [adapter](adapters.md) to use. Possible values: `:attributes, :json, :json_api`. Default: `:attributes`.
+- `automatic_lookup`: Whether serializer should be automatically looked up or manually provided. Default: `true`
```

Comment:
Right now AMS automatically includes the ActionController::Serialization mixin.  What if you could opt out of that and mix it in to controllers as desired. Would that work?  (As opposed to passing in `adapter: false` in most of your controllers)